docs/.vitepress/config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  base: '/v4/',
```

Comment:
@oskardudycz As per earlier threads of conversations, Jeremy wanted the docs urls to be https://martendb.io and https://martendb.io/v4/ corresponding to 3.x and 4.x docs. Hence I am not sure why you have removed the `base: '/v4/'`.

Reply:
As I wrote in the PR description because it does not work for prod build. As we get closer to completing the v4 docs, I’ll send a follow-up PR adding publishing both to the same deployment.

Reply:
Got you.